content/posts/newsletter/2020-07-01.md

```diff
@@ -0,0 +1,155 @@
+---
+title: TBA - 1 July 2020
+save_as: 2020/07/newsletter-01-july/index.html
+url: 2020/07/newsletter-01-july/
+date: 2020-07-01 00:00
+author: emus
+blog_id: newsletter
+category: Newsletter
+---
+
+Welcome to the XMPP newsletter covering the month of June 2020.
+
+## Subscribe to the newsletter
+
+We suggest you [Subscribe](https://tinyletter.com/xmpp) to receive the next editions in your inbox as soon as it is published! Promote this newsletter to whoever may be interested.
+
+## Help us to build the newsletter
+
+We started drafting every new issue in the [XSF Github repository](https://github.com/xsf/xmpp.org/pulls). 
+And we are always happy to welcome contributors. Just join the discussion in our [Comm-Team group chat (MUC)](xmpp:commteam@muc.xmpp.org?join) and thereby help us sustain this as a community effort. 
+
+You have a project and (want to) write about it? Please consider sharing your news or events here, and promote it to a large audience! Even if you can only spend a few minutes, these would already be helpful.
+Tasks which need to be done on a regular basis are for example:
+- Aggregation of news in the XMPP universe
+- Short formulation of news and events
+- Summary of the monthly communication on extensions (XEP)
+- Review of the draft
+- Translations: especially French, German and Spanish
+
+### Newsletter translations
+
+Translations of the XMPP Newsletter will be released here:
+
+- [German](https://www.jabber.de/)
+- [French](https://linuxfr.org/tags/xmpp/public)
+- [Spanish](https://www.jabber.de/)
+
+## XSF Announcements
+
+JC Brand was so nice to create a [XMPP job board](https://xmpp.work/)! You are able to rewiew jobs offers as well as advertise XMPP services, too. Please take a look, post your own offers and recommend it. XMPP works!
+
+## Article
+
+Ingo Jürgensmann wrote an article on the backround of the "Jabber" and "XMPP" terminology and their [differences and similarities](https://blog.windfluechter.net/content/blog/2020/06/09/1758-jabber-vs-xmpp).
+
+## Video
+
+:es: Conociendo XMPP - The Comunidad Latina de Tecnologías Libres had an online introduction to XMPP for users back in April 2020 - we would like to reference this even if it is a bit late: https://www.youtube.com/watch?v=VwnzJ9V732E
+They also have a user chat with this short and handy MUC ID :) 
+comunidadlatinadetecnologiaslibres@chat.disroot.org 
+
+Daniel Gultsch and Holger Weiss had a virtual talk on their A/V implementation in Conversation and describe the technical details. eta finalized the session with a second talk by defining love by this formula: Asteriks + XMPP = <3
+https://www.youtube.com/watch?v=v_riuu4XU6Q
+
+## Software releases
```

Comment:
Hope it's ok to drop my news about JSXC as a comment.

[JSXC](https://jsxc.org) released [version 4.1.1](https://github.com/jsxc/jsxc/releases/tag/v4.1.1) which fixes several video related issues and enhances the interoperability with Conversations.